exp/etcdrestore/config/default/kustomization.yaml

```diff
@@ -1,20 +1,14 @@
 # Adds namespace to all resources.
 namespace: rancher-turtles-system
 
-namePrefix: rancher-turtles-
+namePrefix: rancher-turtles-etcdbackuprestore
```

Comment:
```suggestion
namePrefix: rancher-turtles-etcdsnapshotrestore
```

Reply:
Nit: We need to use the same CRD name as in the proposal throughout the code (Tiltfile, manager_image_patch.yaml etc), but that can be a follow-up PR.